Update Project is updated by using given feedbacks

ozovalihasan opened the pull request:
-HTML file is checked by https://validator.w3.org/#validate_by_upload and solved all problem
-CSS file is checked by using https://github.com/microverseinc/linters-config
-Commits are made descriptive
-README file is fixed
-.gitignore file added
-I completed project alone because my partner withdrew

ozovalihasan committed:
Add All files of https://github.com/ozovalihasan/youtubeCloneByHasan.
TSE of Microverse gave a feedback about branches. Now, "master" branch have only README.md file, and other files are added as a part of new branch.

ozovalihasan committed:
Add .gitignore file is added.

ozovalihasan committed:
Fix CSS file fixed.
Errors found using CSS-linters (https://github.com/microverseinc/linters-config/tree/master/css)

ozovalihasan committed:
Fix CSS file fixed.
Errors found using CSS-linters (https://github.com/microverseinc/linters-config/tree/master/css) are fixed.

ozovalihasan committed:
Add .stickler.yml and stylelint.config.js are added.
To set-up stickler, these files are needed

ozovalihasan committed:
Fix html element of index.html is fixed
There is a warning as a result of https://validator.w3.org/nu/#file. "lang" attribute was missed, it is added

ozovalihasan committed:
Fix Like bar button is fixed.
There is error shwon by https://validator.w3.org/ about button. So, like bar button fixed as having 5 separate button

ozovalihasan committed:
Update Margin are added to left and right of video.

TSE gave a feedback about margin of left and right of video.

ozovalihasan committed:
Document style.css file is moved into CSS folder created
To reset page it is necessary to create another css file. So, all css file will be in the same folder.

ozovalihasan committed:
Add reset.css file is created
To reset browser settings reset.css is needed (https://speckyboy.com/good-bad-css-practices/)

ozovalihasan committed:
Fix Position of navigation bar is fixed.
After using reset.css, it is necessary to fix position of navigation bar

ozovalihasan committed:
Fix Position of video and sidebar is fixed
Navigation bar was on top of video and sidebar.

ozovalihasan committed:
Update Screenshot of cloned Youtube page is updated

ozovalihasan committed:
Fix reset.css file fixed
reset.css file is checked by https://github.com/microverseinc/linters-config. Errors are fixed.

mikenath223 commented:
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

mikenath223 reviewed (changes requested):
### Great work @ozovalihasan  👏👏

I must commend the fact that you continued working solo on this and still got it ready for review within 24hours.
**This is commendable👏👌**

However, a few changes need to be made before we proceed:👇

- [ ] Required - Kindly update your readme with the correct live link consider using this [tool](https://raw.githack.com/) to create a live version link.

- [ ] Required - Kindly ensure you use this [link](http://archive.today/Bss88) to work on this project as stated in the Microverse Adjustment section in Partwright [Screenshot]👇
![ice_screenshot_20200326-124038](https://user-images.githubusercontent.com/33323077/77655272-777f2e80-6f72-11ea-9e98-a5990356a57c.png)

- [ ] Required - In the above screenshot, you will notice that there is a body padding to ensure that elements don't stick to the page sides.

- [ ] Required - Also as the link will indicate, you will need to add a footer section to your page and also add more videos to the aside video section of your page.

- [ ] Optional - This is marked as optional because CSS linting is optional in this project. But in future projects avoid the use of ID selectors for styling. Stick with using just class selectors.

- [ ] Optional - Also consider putting all images in a single folder, helps to keep your project directory clean.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

ozovalihasan committed:
Document All image files moved into a file.
Images aren't seen very well in root file. They are moved into "images" folder

ozovalihasan committed:
Update Project is updated by help of feedback of TSE

Padding are added to left and right of video and sidebar
Navigation bar and information of video are arranged by using http://archive.vn/Bss88 as a source
Added more suggested video into sidebar
Footer is added

ozovalihasan committed:
Update Position of icons in nav bar is arranged

ozovalihasan committed:
Fix Errors shown by html and css linters are fixed

ozovalihasan committed:
Update README file is updated.

Title of project is changed.
Live demo link is changed, but it is directed to mainParts branch because master branch doesn't have index.html file.
